src/graphnet/models/rnn/node_rnn.py

```diff
@@ -0,0 +1,85 @@
+"""Implementation of the NodeTimeRNN model.
+
+(cannot be used as a standalone model)
+"""
+import torch
+
+from graphnet.models.gnn.gnn import GNN
+from graphnet.utilities.config import save_model_config
+from torch_geometric.data import Data
+from typing import Optional
+
+from graphnet.models.components.embedding import SinusoidalPosEmb
+
+
+class Node_RNN(GNN):
+    """Implementation of the RNN model architecture.
+
+    The model takes as input the typical DOM data format and transforms it into
+    a time series of DOM activations pr. DOM. before applying a RNN layer and
+    outputting the an RNN output for each DOM. This model is in it's current
+    state not intended to be used as a standalone model. Furthermore, it needs
+    to be used with a time-series dataset and a "cutter" (see
+    NodeAsDOMTimeSeries), which is not standard in the graphnet framework.
+    """
+
+    @save_model_config
+    def __init__(
+        self,
+        nb_inputs: int,
+        hidden_size: int,
+        num_layers: int,
+        RNN_dropout: float = 0.5,
```

Comment:
i'd recommend just calling this "dropout" as you only have one dropout method here, as far as I can see. You can clarify in the doc string that this dropout is used for the RNN

Reply:
I feel like it makes it more obvious at a glance to have it named **RNN_dropout**. Considering future models that might have dropout layers in different sub-modules then we would have to resort to this kind of naming convention anyways.

Reply:
Alright, but then lets rename it to `rnn_dropout` to be consistent with the snake_case naming scheme in graphnet.